backend/minors/ROB.yaml

```diff
@@ -0,0 +1,117 @@
+---
+type: Minor
+name: Robotics
+code: ROB
+urls:
+- https://robo.princeton.edu/minor
+description: The minor in robotics seeks to provide students from a 
+  diverse range of majors an opportunity to gain an appreciation for these 
+  challenges, develop an understanding of the current state of the art, 
+  explore the multidisciplinary and interdisciplinary nature of robotics, 
+  consider issues of societal significance, and put these lessons into 
+  practice through project-based assignments in courses and through an 
+  independent work project or thesis.
+contacts:
+- type: Director
+  name: Szymon Rusinkiewicz
+  email: smr@princeton.edu
+- type: Program Administrator
+  name: Mitra Kelly
+  email: mkelly@princeton.edu
+req_list:
+  - name: Prerequisites
+    min_needed: ALL
+    max_counted: 1
+    explanation: The minor is open to students from any major who have 
+      a satisfactory background in mathematics, physics, and computing.  
+      Students should have successfully completed or placed out of 
+      mathematics, introductory physics (including mechanics), and 
+      intoductory computer science.
+    req_list:
+    - name: Multivariable Calculus
+      max_counted: 1
+      min_needed: 2
+      explanation:
+      course_list:
+      - MAT 201
+      - MAT 202
+      - MAT 203
+      - MAT 204
```

Comment:
We should split the math reqs into two subreqs, one for Linear and one for multi